extensions/amp-flying-carpet/0.1/amp-flying-carpet.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CSS} from '../../../build/amp-flying-carpet-0.1.css';
+import {Layout} from '../../../src/layout';
+import {isExperimentOn} from '../../../src/experiments';
+import {dev, user} from '../../../src/log';
+
+/** @const */
+const EXPERIMENT = 'amp-flying-carpet';
+
+/** @const */
+const TAG = 'amp-flying-carpet';
```

Comment:
@dvoytenko now that tag and experiment are the same (previously tag was the class name), can we just go with 1?